app/controllers/krikri/reports_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Krikri
+  class ReportsController < ApplicationController
```

Comment:
I got a little confused by the name of this controller, given that we have a `ValidationReportsController` below. Could this be something like `ValidationReportsListController`, or `ReportsListController`? (The latter is probably better, since we'll have other reports that are not validation reports per se.)